tv-shows-app/src/components/features/auth/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,98 @@
+'use client'
+
+import { mutatorLogin } from "@/fetchers/mutator";
+import { Alert, AlertIcon, Button, FormControl, FormErrorMessage, FormLabel, Heading, Input, Link, Text, chakra } from "@chakra-ui/react";
+import { useRouter } from "next/navigation";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import useSWRMutation from "swr/mutation";
+import NexLink from "next/link";
+import { swrKeys } from "@/fetchers/swrKeys";
+
+interface ILoginFormInputs {
+	email: string;
+	password: string;
+}
+
+export const LoginForm = () => {
+	const router = useRouter();
+	const {register, formState: { errors }, handleSubmit} = useForm<ILoginFormInputs>();
+	const [globalError, setGlobalError] = useState([]);
+
+	const { trigger } = useSWRMutation(swrKeys.login, mutatorLogin, {
+		onSuccess: (data) => {
+			if (data.status === 'error') {
+				setGlobalError(data.data);
+				return;
+			}
+
+			router.push('/');
```

Comment:
E sad je malo čudno da imamo onSuccess i da gledamo postoji li status error. (Znam da API ima problema) ali ovo je sad nesto sto bi izbjegli...
Htio bih da api pukne ili sam throwati error u boljem formatu da to mogu uhvatiti u onError, ali okej nije presudno.
Ono što bi htio ovdje je nakon login-a, azurirati (mutirati) cache i postaviti svog user-a, a tu ce onda AuthRedirect automatski kickinati i redirectati nas na pocetni page (ne moramo rucno raditi izmijenu rute).